Answers.md

```diff
@@ -0,0 +1,21 @@
+1.  If you saw this HTML: <div class="box box1 box2 box3"></div> which class has the most specificity weight?
+
+They all have the same specifity. If you select it like so: '.box.box1.box2.box3' then it's the most specific.
```

Comment:
They do have all the same specificity but because of that, it will go to the last element! So the answer will be box3.